tailcall-fixtures/fixtures/protobuf/movies.proto

```diff
@@ -27,6 +30,25 @@ message Movie {
     google.protobuf.Timestamp time = 5;
     Genre genre = 6;
 	google.protobuf.Duration duration = 7;
+
+    // Represents the format in which a movie can be released
+    enum MovieFormat {
+        // The movie is released in IMAX format
+        IMAX = 0;
+    }
+    
+    /**
+     * SubMovie reference
+     */
+    SubMovie sub_movie = 8;
+
+    // This is a comment for submovie
```

Comment:
I don't see this comment in generated output, could you check what is the problem?

Reply:
It is getting generated now.